writeengine/bulk/cpimport.cpp

```diff
@@ -982,15 +982,24 @@ void constructTempXmlFile(
     const std::string&       xmlGenTable,
     const std::string&       alternateImportDir,
     const std::string&       S3Bucket,
+    const std::string&       tableOIDStr,
     boost::filesystem::path& sFileName)
 {
     // Construct the job description file name
     std::string xmlErrMsg;
     int rc = 0;
-    std::string tableOIDStr;
-    getTableOID(xmlGenSchema,
-                xmlGenTable,
-                tableOIDStr);
+    std::string localTableOIDStr;
+    if (tableOIDStr.empty())
```

Comment:
This cpimport change reduces the number of syscat requests. These requests are served from the cache though.